proofs/pfedit.ml

```diff
@@ -73,6 +73,7 @@ let solve ?with_end_tac gi info_lvl tac pr =
       | None -> tac
       | Some _ -> Proofview.Trace.record_info_trace tac
     in
+    let nosuchgoal = Proofview.tclZERO (Proof_bullet.SuggestNoSuchGoals (1,pr)) in
```

Comment:
I don't get the point of this first argument if it is always set to 1.

Reply:
>I don't get the point of this first argument if it is always set to 1.

I was incompletely following the previous system.
I'll take some time to think about it.